os-client/src/main/proto/objectstore/mailbox.proto

```diff
@@ -6,8 +6,7 @@ option java_package = "io.provenance.objectstore.proto";
 option java_outer_classname = "Mailboxes";
 
 import "google/protobuf/empty.proto";
-import "google/protobuf/timestamp.proto";
-import "util.proto";
+import "objectstore/util.proto";
```

Comment:
moved these objectstore proto files to sub-folder, to avoid conflict with public_key.proto file, and because I think it makes sense to do so... Since they are in the `objectstore` proto package, they still match the endpoints